faq.md

```diff
@@ -202,6 +202,21 @@ more motion-rich UIs and brand-first designs.
 In order to achieve that level of customized, beautiful design,
 Flutter is architectured to drive pixels instead of the OEM widgets.
 
+### What happens when my mobile OS updates and introduces new widgets?
+
+The Flutter team watches the adoption and demand for new mobile
+widgets from iOS and Android, and aims to work with the community
+to build support for new widgets. This may come in the form
+of lower-level framework features, new composable widgets, or new
+widget implementations. Flutter's
```

Comment:
Should we add a line saying something like 'And because the widgets are compiled into your app, they are then available on any supported OS version' after 'widget implementations'?

Reply:
Sure, but does that help answer the question? Or, maybe that's another answer to "why not oem widgets" ?